features/other_a/profile_edit.feature

```diff
@@ -56,6 +56,9 @@ Scenario: Edit profile - Changing email address and viewing
   When I change my email
     Then I should see "Your email has been successfully updated"
 	And 1 email should be delivered to "bar@ao3.org"
+        And the email should contain "the email associated with your account has been changed to"
+        And the email should contain "valid2@archiveofourown.org"
+        And the email should not contain "translation_missing"
```

Comment:
Oh! The other Ready to Merge emails check for "translation mission" with a space rather than an underscore. Which version is correct?